lib/hammer_cli_foreman_puppet/environment.rb

```diff
@@ -1,8 +1,9 @@
 module HammerCLIForemanPuppet
-  class PuppetEnvironment < HammerCLIForeman::Command
+  class PuppetEnvironment < HammerCLIForemanPuppet::Command
+    include EnvironmentNameMapping
```

Comment:
Loading of the plugin fails with
[ERROR 2021-06-23T09:16:21 Modules] uninitialized constant HammerCLIForemanPuppet::PuppetEnvironment::EnvironmentNameMapping
/home/vagrant/hammer-cli-foreman-puppet/lib/hammer_cli_foreman_puppet/environment.rb:3:in class:PuppetEnvironment